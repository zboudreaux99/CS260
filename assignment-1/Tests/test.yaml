dueDate: 2024-12-06
latePenalties: [10, 20, 100]
test:
  launch: ./lottery
  timelimit: 10
build:
  instructorFiles:
    exclude:
      - lottery.cpp
  studentFiles:
    exclude:
      - makefile
  command: make -k
