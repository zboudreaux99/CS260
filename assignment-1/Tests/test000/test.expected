How many numbers are printed on the lottery ticket? How many numbers are selected in the lottery drawing? 
Your chances of winning are 1 in 6
